components/no-collection-data.js

```diff
@@ -7,14 +8,21 @@ import GlobalContext from "./global-context";
 /**
  * Display a message on a collection page indicating that no viewable collection data exists.
  */
-const NoCollectionData = () => {
+const NoCollectionData = ({ pageTitle = "" }) => {
   const { page } = useContext(GlobalContext);
 
   return (
     <DataPanel className="my-0.5">
-      <div className="text-center italic">No {page.title} to display</div>
+      <div className="text-center italic">
+        No {pageTitle || page.title} to display
```

Comment:
This code doesn't seem to work properly.  The title is always "list items"

Reply:
You mean this, right?
<img width="738" alt="image" src="https://user-images.githubusercontent.com/1181324/215914594-59744415-1c88-402e-8e9f-d544d80a9021.png">
That’s what I intended to display since search-list displays can have a mix of items. Do you think it should display something else?

Reply:
Oh, I thought you will show the name of search item. In that case, it should be OK.